care/emr/migrations/0006_specimen.py

```
@@ -0,0 +1,42 @@
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Specimen',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('external_id', models.UUIDField(db_index=True, default=uuid.uuid4, unique=True)),
+                ('created_date', models.DateTimeField(auto_now_add=True, db_index=True, null=True)),
+                ('modified_date', models.DateTimeField(auto_now=True, db_index=True, null=True)),
+                ('deleted', models.BooleanField(db_index=True, default=False)),
+                ('history', models.JSONField(default=dict)),
```

Ruff (0.8.0)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)